Druid/DRMethodIRGenerator.class.st

```diff
@@ -170,10 +170,12 @@ DRMethodIRGenerator >> visitInstanceVariableNode: aRBVariableNode [
 
 { #category : 'visiting' }
 DRMethodIRGenerator >> visitTemporaryVariableNode: aRBVariableNode [
-
+	
 	| value |
+	controlFlowGraph addTemporaries: aRBVariableNode name.
```

Comment:
I'm wondering if we should not ask for all the temporarios when the method node is visited.

Now, we are doing it lazily (on the usage), what is good in the sense that we can eliminate unused temps.
But what will happen when the only use of the variable is inside a block? 🤔